acai-ios/SectionControllers/DetailViewSectionControllers/ButtonSectionController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  ButtonSectionController.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/7/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import IGListKit
+
+class ButtonSectionController: ListSectionController {
+    var menuItem: MenuItem!
+    var customizationOptions: [OrderCustomizationOption]!
+    override init() {
+        super.init()
+    }
+
+    override func numberOfItems() -> Int {
+        return 2
+    }
+    
+    override func sizeForItem(at index: Int) -> CGSize {
+        guard let context = collectionContext else {
+            return .zero
+        }
+        return CGSize(width: context.containerSize.width, height: 70)
+    }
+        
+    override func cellForItem(at index: Int) -> UICollectionViewCell {
+        let cell = collectionContext!.dequeueReusableCell(of: ButtonCollectionViewCell.self, for: self, at: index)
```

Comment:
you can do:
`let cell = collectionContext.dequeueReusableCell(of: ButtonCollectionViewCell.self, for: self, at: index) as! ButtonCollectionViewCell` so that you don't have to do the check on line 32